src/Tracing/Span.php

```diff
@@ -58,8 +59,9 @@ public function getParentContext(): ?SpanContext
         return $this->parentSpanContext !== null ? clone $this->parentSpanContext : null;
     }
 
-    public function addLinks()
-    {;
+    public function addLinks(): int
+    {
+        return 0;
```

Comment:
What's the semantic for returning `int`?

Reply:
This was just temporary.  I can remove / change it if required.